dali/pipeline/operators/sequence/element_extract.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/sequence/element_extract.h"
+
+namespace dali {
+
+DALI_SCHEMA(ElementExtract)
+    .DocStr(R"code(Extracts one or more elements (frames) from input)code")
```

Comment:
You assume that we operate only on sequences of frames (4D tensors). I think it would be nice to generalize to N-dim tensors.

Reply:
In that case, for N-dim tensors we will always extract according of indexes of the first dimension. Is that right?

Reply:
Yeah, we can treat any tensor with dims [n1 x n2 x ... nk] as sequence of n1 elements with dims [n2 x ... nk]

Reply:
Done